### PR TITLE
Update VKontakteAuthenticationHandler.cs

### PR DESCRIPTION
**Pull request by YuriPetskus**
Email support. VK send it with access_token

### CODE REVIEW

File: src/Owin.Security.Providers.VKontakte/VKontakteAuthenticationHandler.cs [hunk 1]
```diff
@@ -95,6 +95,10 @@ protected override Task ApplyResponseChallengeAsync()
                 var user = await GetUser(response, accessToken);
 
                 var context = CreateAuthenticatedContext(user, accessToken, properties);
+    
+                // Email support. VK send it with access_token
+                context.Identity.AddClaim(new Claim(ClaimTypes.Email, response["email"].ToString(), XmlSchemaString,
```
Comment (TerribleDev) [inline src/Owin.Security.Providers.VKontakte/VKontakteAuthenticationHandler.cs:100]: 2 things...
1. Can this line be moved into the CreateAuthenticatedContext() function?
2. Can we instead get the email out of the response and set it to a property in `VKontakteAuthenticatedContext`, and then use that property in this line (where `response["email"].ToString()` is)

Comment (YuriPetskus) [inline src/Owin.Security.Providers.VKontakte/VKontakteAuthenticationHandler.cs:100]: Of course, if this would make code simple and clear

**Comment by p4rnicke** [comment]
I just saw I made the same changes in the last pull request.
When will this PR get into the master?

**Comment by TerribleDev** [comment]
@p4rnicke it will once the feedback I gave is implemented.

**Comment by TerribleDev** [comment]
@p4rnicke I just fixed it up it will be on nuget shortly as 2.11.1

**Pull request merged**